hack/generate-image-overrides.sh

```
@@ -0,0 +1,51 @@
+if [[ "${ODH_PLATFORM_TYPE:-OpenDataHub}" != "OpenDataHub" ]]; then
+    platform="rhoai"
+fi
+
+env_names=$($YQ eval ".imageOverrides | keys | .[]" "$CONFIG_FILE" 2>/dev/null || true)
+if [[ -z "$env_names" ]]; then
+    echo "No image overrides defined"
+    exit 0
+fi
+
+mkdir -p "$(dirname "$OUTPUT_FILE")"
+> "$OUTPUT_FILE"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)